src/libmongoc/tests/test-mongoc-topology.c

```diff
@@ -40,7 +40,7 @@ checks_cleanup (checks_t *checks)
 static bool
 checks_cmp (checks_t *checks, const char *metric, char cmp, int expected)
 {
-   int actual;
+   int actual = 0;
```

Comment:
Another gcc warning fix. It was complaining about L61 being a potential read of uninitialized "actual". Though that should not happen since L55 aborts.